src/main/java/com/zendesk/maxwell/schema/SchemaStore.java

```diff
@@ -213,15 +222,21 @@ private Database restoreDatabase(int id, String name, String encoding) throws SQ
 		while (tRS.next()) {
 			String tName = tRS.getString("name");
 			String tEncoding = tRS.getString("encoding");
+			String tPKs = null;
+			try {
+				tPKs = tRS.getString("pk");
+			} catch ( SQLException e ) {}
+
```

Comment:
Since the first thing we do when maxwell starts is: migrate maxwell db, capture the table pks, and update the current maxwell tables.. do we need a try block? You will get NULL anyway?